src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -319,6 +319,8 @@ private LoggerConfiguration CreateLoggerConfiguration(IServiceProvider servicePr
                 .MinimumLevel.Debug()
                 .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
                 .Enrich.FromLogContext()
+                .Enrich.WithVersion()
```

Comment:
We should add correlation as well or is this pending a release?

Reply:
See above.

Reply:
It is added below 😄 . See the line `.WithHttpCorrelationInfo`.